package.json

```diff
@@ -37,6 +37,8 @@
   },
   "devDependencies": {
     "babel-eslint": "^3.1.21",
-    "eslint": "^0.24.0"
+    "blue-tape": "^0.1.9",
+    "eslint": "^0.24.0",
+    "tape": "^4.0.0"
```

Comment:
At the moment `tape` is used only to reuse its CLI, until https://github.com/spion/blue-tape/pull/12 is merged